app/controllers/contestant_projects_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ContestantProjectsController < ApplicationController
```

Comment:
Great work using a joins controller here! This is the more RESTful option rather than squeezing in the functionality in the other controllers.